data/mods/innawood/overmap/specials_kept.json

```diff
@@ -0,0 +1,574 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "Strangle Temple",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "temple_stairs" },
+      { "point": [ 0, 0, -1 ], "overmap": "temple_stairs" },
+      { "point": [ 0, 0, -2 ], "overmap": "temple_stairs" },
+      { "point": [ 0, 0, -3 ], "overmap": "temple_stairs" },
+      { "point": [ 0, 0, -4 ], "overmap": "temple_stairs" },
+      { "point": [ 0, 0, -5 ], "overmap": "temple_finale" }
+    ],
+    "locations": [ "forest" ],
+    "city_distance": [ 40, -1 ],
+    "occurrences": [ 10, 100 ],
+    "rotate": false,
+    "flags": [ "WILDERNESS", "UNIQUE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Standing stones",
```

Comment:
I think this might classify as man-made and not something the Cataclysm has caused

Reply:
The idea is that it is sufficiently ambiguous whether these were made by men or by aliens.